src/Services/Kros.Authorization.Api/Application/Services/AuthorizationService.cs

```diff
@@ -52,11 +53,16 @@ public async Task<string> CreateJwtTokenAsync()
             var oidcClaims = await GetOidcUserClaimsAsync();
             var user = await _userService.TryCreateDefaultUserAsync(oidcClaims);
 
-            var allUserClaims = new List<Claim>();
-            allUserClaims.AddRange(oidcClaims);
-            allUserClaims.AddRange(GetUserAuthorizationClaims(user));
+            if (user != null)
+            {
+                var allUserClaims = new List<Claim>();
+                allUserClaims.AddRange(oidcClaims);
+                allUserClaims.AddRange(GetUserAuthorizationClaims(user));
+
+                return JwtAuthorizationHelper.CreateJwtTokenFromClaims(allUserClaims, _apiJwtAuthorizationOptions.JwtSecret);
+            }
 
-            return JwtAuthorizationHelper.CreateJwtTokenFromClaims(allUserClaims, _apiJwtAuthorizationOptions.JwtSecret);
+            return string.Empty;
```

Comment:
Ok, it's demo. But by my opinion is, that in real `AuthorizationService` is not ok returning `string.Empty` if user is not found. I'd throw an exception.

Reply:
I changed it: 
throw new UnauthorizedAccessException(Properties.Resources.AuthorizationServiceForbiddenRequest);